conf/default.cfg

```diff
@@ -132,6 +132,10 @@ use = egg:oioswift#hashedcontainer
 # (equivalent to "ns.flat_bits" namespace parameter).
 #bits=17
 
+# If no authentication and no quota is used
+# you can skip extract of metadata from account and container
+#skip_metadata = false
```

Comment:
I would rewrite this as
```
Skip account and container metadata retrievals.
This can improve performance if this metadata is not used.
Enable only if neither authentication nor quota are enabled.
```